docs/user/overview.md

```diff
@@ -0,0 +1,61 @@
+# Installer Overview #
```

Comment:
nit: we don't use the trailing `#`s for our other docs.  Can we remove them for consistency?